src/cursor.c

```diff
@@ -2043,11 +2043,12 @@ int check_x11_pointer(void) {
 	    if (clipshift) {
 	      static int cnt = 0;
 	      if (x < 0 || y < 0 || x >= dpy_x || y >= dpy_y)  {
-		if (cnt++ % 4 != 0) {
-		  if (debug_pointer)
-		    rfbLog("Skipping cursor_position() outside our clipshift\n");
-		  continue;
-		}
+	        if (cnt++ % 4 != 0) {
+	          if (debug_pointer)
+	            rfbLog("Skipping cursor_position() outside our clipshift\n");
```

Comment:
Well, that's not true anymore, isn't it?

Reply:
It is still true
Behavior before: if cursor out of clipped area then three times of four print "Skipping cursor_position() outside our clipshift" and skip calling `cursor_position(...)`. One time of four - call `cursor_position` as usual.
Behavior now: if cursor out of clipped area then three times of four print "Skipping cursor_position() outside our clipshift" (may be change to "one time of four"?) and every time hide cursor and skip calling `cursor_position`.

As I understand thing "n times of four" is designed to reduce count of "Skipping ..." message in log